client/src/components/games/GamesPerVenue.tsx

```diff
@@ -77,11 +101,15 @@ function GamesPerVenue() {
               </NavLink>
             </div>
           ))}
-          {filteredGames.length === 0 &&
+          {filteredGames &&

```

Comment:
I think this should be || instead. If filteredGames is null, you want to show the placeholder.